serviceWorker.js

```diff
@@ -0,0 +1,83 @@
+// Set to false to disable logging.
+const verbose = true;
+
+async function handleFetch(request) {
+	// Perform the original fetch request and store the result in order to modify the response.
+	const r = await fetch(request).catch(e => console.error(e));
```

Comment:
The catch statement is confusing here. If you use await then you also use try/catch

Reply:
Oversight, will remove it.